### PR TITLE
Ignore prereleases

### PR DESCRIPTION
**Pull request by rtsao**
Fixes https://github.com/fusionjs/fusion-cli/issues/703

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -22,6 +22,11 @@ module.exports = robot => {
   async function handler(context) {
     const release = context.payload.release;
 
+    // Don't generate release notes for prereleases
```
Comment (chrisdothtml) [inline index.js]: I still feel like release notes are handy for pre-releases, no?

**Review by chrisdothtml — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (rtsao) [reply index.js]: I was thinking this would be confusing if changes are listed multiple times

Comment (chrisdothtml) [reply index.js]: Ah nevermind, thought this was referring to prerelease PRs, not the Github releases page. Yeah makes sense to leave out the notes for prereleases

**Comment by rtsao** [comment]
!merge

**Pull request merged**